interview-questions.md

```diff
@@ -12,46 +12,54 @@ Your answer:
 
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? 
 
-Your answer:
+Your answer: git is a method of version control.  Utilizing the commands of git add and git commit, you are able to track the ways in which you modify your code.  GitHub is a cloud based repository in which your code is uploaded and made accessible to others.
 
-Researched answer:
+Researched answer: While both git and GitHub are ways to maintain version control, git functions on the local machine, while GitHub exists in the cloud (remote storage)
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The JavaScript operators that will return a Boolean Value are any of the conditional operators.  These include < (less than), <= (less than or equal to), > (greater than), >= (greater than or equall to), != (not equal to), == or === (loose or strict interpretation of a value),
+&& (checking to see if two conditions are both true), or || (checking to see if one of two given conditions is true).
 
-Researched answer:
+Researched answer: The operators I listed above are correct, in that they will return a Boolean value.  I also found that there were additional conditional operators I had missed, such as !== (not equal type or equal value) and ? (ternary operator). 
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is the value of a particular item in a variable.  Index starts counting at 0, so the answer will always be one less than the location of the item.  Example:  var cats = ("calico", (persian), (manx))
+          console.log(cats[1])
+The above example will output persian to the screen, because it is located at index 1 of the variable.
+    The difference between index and value is that the index will always be one less than the location of a particular value in a given variable.  The value is a specific location, while the index is where it is located in the variable.
 
-Researched answer:
+Researched answer:  I understand the concepts involved in index and value, and determined my response was accurate without further research adding to my understanding.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: Arrays are groups of values taken as a whole, while strings are actually self-contained arrays.  A string is taken as a whole, while the array would be seperated by a , in its defining.
+        Example:  array cats = ["calico", "persian", "manx"] (This provides three values to the array of cats)
+                  string cats = "calico" (This provides an array consisting of 6 values, each of which may be called) 
 
-Researched answer:
+Researched answer: My research showed me that any given variable by itself is indeed an array, but an array dataType must be defined using the [ ] brackets.
 
```

Comment:
The syntax for a string ---> "string"
A string is one value surrounded by quotation marks.
The syntax for an array ---> [ "string1" , "string2", 3, 4]
An array is a collection of different values surrounded by square brackets. Each value is separated by a comma aka comma separated values (CSV). 
Here are some references in the syllabus. 
string: https://github.com/learn-academy-2022-hotel/Syllabus/blob/main/javascript/intro.md
array: https://github.com/learn-academy-2022-hotel/Syllabus/blob/main/javascript/arrays.md